### PR TITLE
Benchmark extrinsics

### PR DESCRIPTION
**Pull request by SaMotlagh**
- removed benchmark extrinsics with no origin
- removed benchmarking test file including tests for those benchmarking calls.

this PR addresses BIT 307.

### CODE REVIEW

**Review by S0AndS0 — APPROVED** [review]

File: pallets/subtensor/src/lib.rs [hunk 1]
```diff
@@ -1536,42 +1536,11 @@ pub mod pallet {
 			Self::do_set_total_issuance(origin, total_issuance)
 		}
 
-		#[pallet::call_index(50)]
+		#[pallet::call_index(47)]
```
Comment (camfairchild) [inline pallets/subtensor/src/lib.rs]:
```suggestion
		#[pallet::call_index(50)]
```

We shouldn't change the call indices for existing calls

**Review by camfairchild — CHANGES_REQUESTED** [review]
One change

File: pallets/subtensor/src/lib.rs (hunk 1 above)
Comment (SaMotlagh) [reply pallets/subtensor/src/lib.rs]: huum, right. fixed.

**Review by camfairchild — APPROVED** [review]
LGTM 🔥🔥🔥

**Pull request merged**